flexget/plugins/filter/crossmatch.py

```diff
@@ -94,10 +97,31 @@ def entry_intersects(self, e1, e2, fields=None):
             log.trace('v1: %r' % v1)
             log.trace('v2: %r' % v2)
 
-            if v1 == v2:
-                common_fields.append(field)
+            if mode == "flexible":
+                v1Re = v1.lower()
+                v1Re = unicodedata.normalize('NFKD', v1Re).encode('ASCII', 'ignore')
+                v1Re = re.sub("[^a-zA-Z0-9 ]", " ", v1Re)
+                v1Re = v1Re.replace(' ','.*')
```

Comment:
Isn't this going to be way too way too greedy? `Spider.*Man` would match both "Spider-Man" and "Spiders are the bane of man". It also strikes me as a bad idea to treat all non-alphanumeric characters as noise. This is going to cause a lot of false positives for non-English entries.